packages/eslint-plugin/src/stylex-sort-keys.js

```diff
@@ -9,6 +9,7 @@
 
 'use strict';
 
+import getSourceCode from './utils/getSourceCode';
```

Comment:
nit: we typically import modules after importing types. see the other util imports below